DigitalExcellence/dex-frontend
# Project wizard improvements

## Description

While testing the wizard we ran into some small issues, this pull request addresses those issues.

## Type of change


## Checklist


## Steps to Test or Reproduce

Steps to reproduce the behavior:
When adding a collaborator using the project wizard the role will always be "Developer":
1. Open project wizard
2. Add collaborator with a role not being "Developer". 
3. Go to next step. Go back to collaborator step.
4. Collaborator is now developer.

Source: "Add project manually" should be first option.
1. https://staging.dex.software/project/add
2. First option is disabled Gitlab source.

Update after delete project
1. Delete a project
2. Project still shows on the project overview until you refresh 

Icon on the first wizard page (GitLab & GitHub)
1. Open GitLab / GitHub wizard.
2. On the first step a placeholder is shown.

## Link to issue

Closes: #462
Closes: #460 
Closes: #492 
Closes: #490
